main.go

```diff
@@ -170,3 +175,33 @@ func main() {
 		e.Logger.Fatal(err)
 	}
 }
+
+func createRateLimitMiddleware(seconds int, burst int) echo.MiddlewareFunc {
+	config := middleware.RateLimiterMemoryStoreConfig{
+		Rate:  rate.Every(time.Duration(seconds) * time.Second),
+		Burst: burst,
+	}
+	return middleware.RateLimiter(middleware.NewRateLimiterMemoryStoreWithConfig(config))
```

Comment:
if i just pass rate.Every(10 * time.Second) that is same as 0.1 and it is treated as 0
only way to do this it seems is to set rate to 0.1, and burst to 1, and it is acting as 1 every 10 seconds

tested it using postman with /create endpoint